easybuild/easyconfigs/l/LWP-Protocol-https/LWP-Protocol-https-6.07-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,72 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Bundle'
+
+name = 'LWP-Protocol-https'
+version = '6.07'
+
+homepage = 'https://github.com/libwww-perl/lwp-protocol-https'
+description = "Provide https support for LWP::UserAgent"
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+builddependencies = [('binutils', '2.32')]
+dependencies = [
+    ('Perl', '5.30.0'),
+]
+
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perldoc -lm %(ext_name)s ", "")
+
+exts_list = [
+    ('LWP::UserAgent', '6.06', {
```

Comment:
This is also included as extension in `Bio-EUtilities`, why not move this into `Perl` instead (together with any other extensions that are generic, i.e. don't require any additional dependencies)?